onchange_helper/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Florian da Costa <florian.dacosta@akretion.com>
+* Andrea Stirpe <a.stirpe@onestein.nl>
```

Comment:
@sbejaoui You can add yourself into the contributors list

Reply:
Done